packages/yoastseo/spec/specHelpers/scoring/seoAssessorTests.js

```diff
@@ -41,40 +41,41 @@ export function checkAssessmentAvailability( assessor, isProductAssessor = false
 		defaultAssessments.pop();
 	}
 
-	let requiresTextAssessments = [ "images", "externalLinks", "internalLinks" ];
+	let commonAssessments = [ "images", "externalLinks", "internalLinks" ];
 	if ( isCollection || isStoreBlog || isTaxonomy ) {
-		requiresTextAssessments = [];
+		commonAssessments = [];
 	}
 	if ( isProduct ) {
-		requiresTextAssessments = [ "images" ];
+		commonAssessments = [ "images" ];
 	}
 
 	const requiresKeyphraseAssessments = [ "introductionKeyword" ];
 	if ( isStoreBlog ) {
 		requiresKeyphraseAssessments.pop();
 	}
 
-	const mostAssessments = [ ...defaultAssessments, ...requiresTextAssessments, ...requiresKeyphraseAssessments ];
+	const textAssessments = [ ...defaultAssessments, ...commonAssessments ];
+	const mostAssessments = [ ...textAssessments, ...requiresKeyphraseAssessments ];
 
 	it( "runs assessments without any specific requirements", function() {
 		const paper = new Paper( "" );
 		const assessments = assess( paper );
 
-		expect( assessments.sort() ).toEqual( defaultAssessments.sort() );
+		expect( assessments.sort() ).toEqual( textAssessments.sort() );
 	} );
 
 	it( "additionally runs assessments that only require a text", function() {
 		const paper = new Paper( "text" );
 		const assessments = assess( paper );
 
-		expect( assessments.sort() ).toEqual( [ ...defaultAssessments, ...requiresTextAssessments ].sort() );
+		expect( assessments.sort() ).toEqual( textAssessments.sort() );
```

Comment:
I think this can be removed? These assessments don't require text so adding text doesn't change anything compared to the previous scenario.

Reply:
Correct, I will remove this test! 👍